hw/ip/pwm/dv/pwm_sim_cfg.hjson

```diff
@@ -46,11 +46,10 @@
 
   // List of test specifications.
   tests: [
-// TODO : for V1
-//    {
-//      name: pwm_smoke
-//      uvm_test_seq: pwm_smoke_vseq
-//    }
+    {
+      name: pwm_smoke
+      uvm_test_seq: pwm_smoke_vseq
+    }
```

Comment:
Please enable PWM in smoke regr below. Please add PWM to our CI and nightly if ready. Also enable dv linting.

Reply:
Hi Tung, you merged this PR without addressing this. Please address in a followup PR.

Reply:
@sriyerg that is on me.
@tunghoang290780 and I will get this fixed.